### ISSUE TITLE
'https://colorjs.io/docs/spaces' doesn't always load the list of color spaces

### ISSUE THREAD

**Issue by ilikescience** [color-js/color.js#502]
When navigating directly to https://colorjs.io/docs/spaces, the list of color spaces doesn't always load. I get inconsistent error messages, but it's usually something like `Unhandled Promise Rejection: ReferenceError: Can't find variable: Mavo`.

**Comment by facelessuser** [comment]
Yeah, there are some documentation issues currently. Code blocks are also not editable currently which makes the `Play!` tab unusable.  Some time needs to be spent on the docs to get it back into shape.

**Comment by scriptype** [comment]
I don't know if this is related but sometimes the https://colorjs.io doesn't open for me, and so `/dist/color.js` also becomes unavailable.
